basher.go

```diff
@@ -232,10 +233,23 @@ func (c *Context) Run(command string, args []string) (int, error) {
 	for _, arg := range args {
 		argstring = argstring + " '" + arg + "'"
 	}
+
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals)
```

Comment:
do we need to worry about this channel getting left around? i tried a `defer close(signals)` but that seemed to close it down prematurely....